test/test_helper.rb

```diff
@@ -32,6 +32,7 @@ def spoom_gemfile
         gemspec name: "spoom", path: "#{SPOOM_PATH}"
         gem "tapioca"
         gem "sorbet-static-and-runtime", "#{Sorbet::GEM_VERSION}"
+        gem "json", "2.7.2"
```

Comment:
There are 75 failures due to `/opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/3.3.0/json/common.rb:3: warning: ostruct was loaded from the standard library, but will no longer be part of the default gems starting from Ruby 3.5.0.` so I opted for a general solution for all tests.